src/connection.ts

```diff
@@ -428,6 +427,7 @@ export class Connection {
                   if (typeof parsedBody !== "string") {
                     parsedBody = res.body.toString("utf-8");
                   }
+                  // @ts-ignore
```

Comment:
why do we use ts-ignore here and also can you add the rule in the comment after ts-ignore

Reply:
@dlozina-macrometa I think we can do catch(e: unknown) to get rid of unknown errors from typescript or if you want to ignore them then you can set it in tsconfig.json under compilerOptions.
This link captures everything related to this unknown type error and how to deal with it:
https://bobbyhadz.com/blog/typescript-object-is-of-type-unknown#:~:text=The%20%22Object%20is%20of%20type,(err%20instanceof%20Error)%20%7B%7D%20.

Reply:
Yes, we can do that, but maybe the best solution would be to find out what type of error this is, we know it contains a `response`, otherwise, it will be undefined. '@ts-ignore' is a placeholder for refactoring.